### PR TITLE
remove bash array typing

### PR DESCRIPTION
**Pull request by eriksywu**

### CODE REVIEW

**Review by xuto2 — APPROVED** [review]

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 1]
```diff
@@ -214,15 +214,15 @@ cleanUpImages() {
     export targetImage
     function cleanupImagesRun() {
         {{if NeedsContainerd}}
-        images_to_delete=$(ctr --namespace k8s.io images list | grep -vE "${KUBERNETES_VERSION}$|${KUBERNETES_VERSION}.[0-9]+$|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep ${targetImage} | awk '{print $1}')
+        images_to_delete=$(ctr --namespace k8s.io images list | grep -vE "${KUBERNETES_VERSION}|${KUBERNETES_VERSION}.[0-9]+|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep ${targetImage} | awk '{print $1}' | tr ' ' ' ')
```
Comment (eriksywu) [inline parts/linux/cloud-init/artifacts/cse_install.sh:217]: need to figure out why read doesn't seem to work without 'tr ' ' ' '' `

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 2]
```diff
@@ -214,15 +214,15 @@ cleanUpImages() {
     export targetImage
     function cleanupImagesRun() {
         {{if NeedsContainerd}}
-        images_to_delete=$(ctr --namespace k8s.io images list | grep -vE "${KUBERNETES_VERSION}$|${KUBERNETES_VERSION}.[0-9]+$|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep ${targetImage} | awk '{print $1}')
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_install.sh:217]: why do we need to remove the trailing $

File: parts/linux/cloud-init/artifacts/cse_install.sh [hunk 3]
```diff
@@ -214,15 +214,15 @@ cleanUpImages() {
...
+        images_to_delete=$(ctr --namespace k8s.io images list | grep -vE "${KUBERNETES_VERSION}|${KUBERNETES_VERSION}.[0-9]+|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep ${targetImage} | awk '{print $1}' | tr '\n' ' ')
         {{else}}
-        images_to_delete=$(docker images --format '{{OpenBraces}}.Repository{{CloseBraces}}:{{OpenBraces}}.Tag{{CloseBraces}}' | grep -vE "${KUBERNETES_VERSION}$|${KUBERNETES_VERSION}.[0-9]+$|${KUBERNETES_VERSION}-|${KUBERNETES_VERSION}_" | grep ${targetImage})
```
Comment (xuto2) [inline parts/linux/cloud-init/artifacts/cse_install.sh:219]: the same here.

**Comment by eriksywu** [comment]
addresses @AKS#650

File: parts/linux/cloud-init/artifacts/cse_install.sh (hunk 2 above)
Comment (eriksywu) [reply parts/linux/cloud-init/artifacts/cse_install.sh:217]: reverted - bad copy/paste from a wip script I was trying out on an actual aks base VM

**Pull request merged**